srcpkgs/rpi-kernel/template

```diff
@@ -12,22 +12,22 @@
 #
 # WARNING: keep all rpi*-kernel packages in sync
 
-_githash="82c6f8643398e222099066bfffb2070772f0696f"
+_githash="64ad74084fa44abe8689564071df5729ded4c589"
 _gitshort="${_githash:0:7}"
 
 pkgname=rpi-kernel
-version=5.10.110
+version=5.15.61
 revision=1
 archs="armv6l*"
 wrksrc="linux-${_githash}"
-hostmakedepends="perl kmod uboot-mkimage openssl-devel bc bison flex"
+hostmakedepends="perl kmod uboot-mkimage openssl-devel bc bison flex xz"
 makedepends="ncurses-devel"
 maintainer="Piraty <piraty1@inbox.ru>"
 homepage="http://www.kernel.org"
 license="GPL-2.0-only"
 short_desc="Linux kernel for Raspberry Pi zero/1 (${version%.*} series [git ${_gitshort}])"
 distfiles="https://github.com/raspberrypi/linux/archive/${_githash}.tar.gz"
-checksum=f3ee7782c90ecb3e468e89602248a1de45b466d5d4c1af4748f7ca088c0fcab3
+checksum=64ad74084fa44abe8689564071df5729ded4c589
```

Comment:
checksum doesn't matcht

Reply:
I seem to have mis-pasted something here, sorry